Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -360,15 +361,39 @@ - (void)textInputDidEndEditing
                                  reactTag:self.reactTag
                                      text:[self.backedTextInputView.attributedText.string copy]
                                       key:nil
+                               iosKeyCode:nil
                                eventCount:_nativeEventCount];
 
   [_eventDispatcher sendTextEventWithType:RCTTextEventTypeBlur
                                  reactTag:self.reactTag
                                      text:[self.backedTextInputView.attributedText.string copy]
                                       key:nil
+                               iosKeyCode:nil
                                eventCount:_nativeEventCount];
 }
 
+- (void)pressesBegan:(NSSet<UIPress *> *)presses withEvent:(UIPressesEvent *)event{
+  UIPress* firstPress = [[presses allObjects] objectAtIndex:0];
+  
+  if (@available(iOS 13.4, *)) {
+    if(firstPress != nil){
+      UIKey* key = firstPress.key;
+      CFIndex keyCode = (CFIndex) key.keyCode;
+      NSInteger *iosKeyCode = (NSInteger*) keyCode;
+      
+      if(key != nil){
```

Comment:
If key is nil, wouldn't `CFIndex keyCode = (CFIndex) key.keyCode;` get exceptions already?

Reply:
Good point - if key actually happens to be null that is of course the case.